README.md

```diff
@@ -28,14 +28,7 @@ Try it! It’s fun, we promise :)
 
 By the way, creating issues is fun too!
 
----
```

Comment:
What "Andrii" did to you, so you decided to remove his lines?

Reply:
I removed lines "Andrii"